hatch/settings.py

```diff
@@ -19,8 +20,8 @@
         ('build', ''),
     ])),
     ('pypi_username', ''),
-    ('name', 'U.N. Owen'),
-    ('email', 'me@un.known'),
+    ('author', get_user() or 'U.N. Owen'),
```

Comment:
I understand the decision here, but can we please keep it as before? It would be a breaking change.